### PR TITLE
chore(ci): always enable actions on PR

### PR DESCRIPTION
**Pull request by sauyon**
Noticed there's no CI on #2645; let's just have actions run on every PR?

### CODE REVIEW

**Review by aarnphm — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,10 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request:
-    branches:
-      - main
-      - runners-1.0
+  pull_request
```
Comment (aarnphm) [inline .github/workflows/ci.yml:7]:
```suggestion
  pull_request:
```

Comment (sauyon) [reply .github/workflows/ci.yml:7]: I think we want it to just be a string if we're not configuring it.

Comment (aarnphm) [reply .github/workflows/ci.yml:7]: can you link it to the action documentation, i can't seem to find it

Comment (aarnphm) [reply .github/workflows/ci.yml:7]: iirc 
```yaml
on: [push, pull_request]
``` would do the trick
so what you are having is probably right.

**Pull request merged**